pages/ecosystem.js

```diff
@@ -15,17 +16,28 @@ const Ecosystem = ({ readme, sidebarPages }) => (
     description="Ecosystem of styled-components"
   >
     <p>
-      This is an incomplete list of awesome things built with styled-components. If you have something to share, please add it to the <Link href="https://github.com/styled-components/awesome-styled-components" inline>awesome-styled-components</Link> repo on GitHub and it will automatically show up here!
+      This is an incomplete list of awesome things built with styled-components.
+      If you have something to share, please add it to the{' '}
+      <Link
+        href="https://github.com/styled-components/awesome-styled-components"
+        inline
+      >
+        awesome-styled-components
+      </Link>{' '}
+      repo on GitHub and it will automatically show up here!
```

Comment:
Prettier does a not very good job of formatting jsx in 1.14 – see https://github.com/prettier/prettier/issues/4941

Perhaps using prettier from master on github is a better idea as it should not create such a mess and it will also support mdx